fastify.js

```diff
@@ -436,6 +436,9 @@ function build (options) {
       preHandler.push.apply(preHandler, opts.preHandler || _fastify._hooks.preHandler)
```

Comment:
Correct If I'm wrong but why we add the prehandler twice. Here and after the loop.

Reply:
I've edited the test in order to check is the `beforeHandler` is executed twice

Reply:
My mistake it's different.